buildingspy/development/regressiontest.py

```diff
@@ -7,6 +7,14 @@
 #######################################################
 from __future__ import division
 from __future__ import print_function
+from future import standard_library
```

Comment:
for documentation see 
http://python-future.org/reference.html#module-future.standard_library